tests/spec/shell/cat.spec.js

```diff
@@ -1,20 +1,25 @@
 var Filer = require('../../..');
 var util = require('../../lib/test-utils.js');
 var expect = require('chai').expect;
+var optionals = require('../../../src/shell/optional');
 
-describe('FileSystemShell.cat', function() {
+describe('(optional) FileSystemShell.cat', function() {
```

Comment:
I'll pull them. It's more a sign of my confusion about where the optional stuff will live when we split the shells off.